key-manager-oapi.yaml

```diff
@@ -0,0 +1,223 @@
+openapi: "3.0.3"
+
+info:
+  title: "Eth2 key manager API"
+  description: |
+    API specification for a key manager client, which enables users to manage keystores.
+
+    The key manager API is served by the binary holding the validator keys. This binary may be a remote signer or a validator client.
+
+    All routes SHOULD be exposed through a secure channel, such as with HTTPs, an SSH tunnel, a VPN, etc.
+
+    All requests by default send and receive JSON, and as such should have either or both of the "Content-Type: application/json"
+    and "Accept: application/json" headers.
+
+    All sensitive routes are to be authenticated with a token. This token should be provided by the user via a secure channel:
+      - Log the token to stdout when running the binary with the key manager API enabled
+      - Read the token from a file available to the binary
+  version: "Dev - Eth2Spec v1.0.1"
+  contact:
+    name: Ethereum Github
+    url: https://github.com/ethereum/eth2.0-apis/issues
+  license:
+    name: "Apache 2.0"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+
+servers:
+  - url: "{server_url}"
+    variables:
+      server_url:
+        description: "key manager API url"
+        default: "https://public-mainnet-node.ethereum.org"
+
+paths:
+  /eth/v1/keystores:
+    get:
+      operationId: ListKeys
+      summary: List Keys.
+      description: |
+        List all validating pubkeys known to and decrypted by this keymanager binary
+      security:
+        - cookieAuth: []
+      responses:
+        "200":
+          description: Success response
+          content:
+            application/json:
+              schema:
+                title: ListKeysResponse
+                type: object
+                properties:
+                  keys:
```

Comment:
I think this might be normally `data` in all the others?

Reply:
I think if we have a more specific term we can use then we should use it, e.g. delete and insert return `statuses` at the top-level. However `keys` doesn't match perfectly in this case, so maybe `data` is preferable? Could also just do `data` for all of them?

Reply:
on the main beacon-api, post doesn't require the wrapper, only the responses... I think it was in case we need to send extra context outside of the data at some point...